src/components/sponsor-table/columns.tsx

```diff
@@ -0,0 +1,79 @@
+"use client";
+
+import { ColumnDef } from "@tanstack/react-table";
+import { DownloadIcon } from "@radix-ui/react-icons";
+import { Button } from "@/components/ui/button";
+
+export type Reimbursement = {
```

Comment:
You should be able to use the interface specified in the Reimbursement schema now.

Reply:
done